src/content/docs/dropins/checkout/containers/overview.mdx

```diff
@@ -46,3 +46,91 @@ The `Title` property is a handler used to render a customized title content.
 :::note[Checkout containers]
 In the pages that follow, you will learn how to customize the `Title` and how to use it.
 :::
+
+## Active property
+
+The `active` property controls whether the container is active, this means it is visible and responds to system events. When set to _false_, the container is not reactive, therefore it does not subscribe to system events and it's not rendered in the user interface.
```

Comment:
```suggestion
The `active` property indicates whether the container is active. When set to _false_, the container becomes non-reactive, meaning it will not respond to system events, will not subscribe to them, and will not be displayed in the user interface.
```